### PR TITLE
FEAT skeleton components batch 1

### PR DESCRIPTION
**Pull request by MarcFairbrother**
## Description

- skeleton header
- skeleton text field
- skeleton button

-----

image
image
image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2762/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/16506/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2762/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/16509/ui-diff)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/16510/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2762/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2762/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18220/ui-diff)

**Review by Supamiu — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18237/ui-diff)

**Review by jeremie-lucca — DISMISSED** [review]

File: packages/ng/skeleton/skeleton-button/skeleton-button.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="skeleton is-loading" [class.mod-dark]="dark">
+	<div class="button" [class.mod-dark]="dark" inert>
```
Comment (vvalentin-lucca) [inline packages/ng/skeleton/skeleton-button/skeleton-button.component.html:2]: What’s the point of `inert` here if you're on a `div`?

File: packages/ng/skeleton/skeleton-button/skeleton-button.component.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@use '@lucca-front/scss/src/components/button';
+@use '@lucca-front/scss/src/components/skeleton';
+
+.skeleton.is-loading {
+  .button {
+    background: transparent;
```
Comment (vvalentin-lucca) [inline packages/ng/skeleton/skeleton-button/skeleton-button.component.scss:6]:
```suggestion
    background-color: transparent;
```

File: packages/ng/skeleton/skeleton-button/skeleton-button.component.html (hunk 1 above)
Comment (MarcFairbrother) [inline packages/ng/skeleton/skeleton-button/skeleton-button.component.html:2]: I would move the `inert` to the root element of the component but it's not a big deal.

**Review by jeremie-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2762/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18242/ui-diff)

**Pull request merged**